asv_benchmarks/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,107 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the airspeed velocity docs for more information.
+# https://asv.readthedocs.io/en/stable/
+import numpy as np
+from numpy import newaxis
+from scipy.stats import circstd
+from scipy.sparse import coo_matrix
+import scipy.signal as ss
+import warnings
+import arviz as az
+from arviz.stats.stats_utils import _circular_standard_deviation, histogram, stats_variance_2d
+from arviz.kde_utils import _fast_kde, _fast_kde_2d
+
+
+class Hist:
+    params = (True, False)
+    param_names = "Numba"
```

Comment:
This have to be a single element tuple, the string is interpreted as an iterable and only the first letter is used: https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=2274&view=logs&j=273a7ac9-cc40-5d2f-3776-d61d22a0ef9c&t=8fe29edb-6726-5e00-142e-7c337980bc77&l=60

See `Atleast_Nd` for an example